src/NetscriptFunctions.ts

```diff
@@ -988,6 +988,15 @@ export function NetscriptFunctions(workerScript: WorkerScript): NS {
 
       LogBoxEvents.emit(runningScriptObj);
     },
+    closeTail: function (pid?: number): void {
```

Comment:
`pid?: unknown = workerScript.pid`

then call `helper.number`

Reply:
Done, I think. It's not exactly what you suggested, but it matches the way another function right next to it works, and doing what you gave me exactly just gives a syntax error.

Reply:
The question mark in my example should not have been there